pom.xml

```diff
@@ -90,7 +90,7 @@ THE SOFTWARE.
     <changelog.url>https://www.jenkins.io/changelog</changelog.url>
 
     <!-- Bundled Remoting version -->
-    <remoting.version>4.13</remoting.version>
+    <remoting.version>3025.vf64a_a_3da_6b_55</remoting.version>
```

Comment:
This version causes a test failure, because remoting version from the commit includes changes for https://github.com/jenkinsci/jenkins/commit/5f9100286be16c434e973efc453a0b35ceee9342 which are not present here. Is there are different release of remoting we can use, or any recommendations how to deal with the test failure?

Reply:
```suggestion
    <remoting.version>4.13.2</remoting.version>
```

Reply:
Nice, thanks. but there's one test failure left for some reason 🤔